test/fares/fares_test.exs

```diff
@@ -1,7 +1,12 @@
 defmodule FaresTest do
   use ExUnit.Case, async: true
   doctest Fares
-  alias Routes.Route
+
+  import Mox
+
+  alias Test.Support.Factories.{Routes.Route, Stops.Stop}
```

Comment:
suggestion (non-blocking): I try not to nitpick style in general but I've come to believe in the utility of this Credo check: https://hexdocs.pm/credo/Credo.Check.Consistency.MultiAliasImportRequireUse.html because splitting up the aliases instead of grouping them by root (as here) makes it easier to grep for module usage in the codebase. Thoughts?